docs/configuration.md

```diff
@@ -326,3 +326,9 @@ This parameter configures the minimum zoom level available. Set it to a not-null
 _Default: Infinity_
 
 This parameter configures the maximum zoom level available. Set it to a lower value to prevent your users from zooming in too deeply.
+
+### removeOnResize
```

Comment:
I would remove this configuration parameter. Instead, I would expose a `destroy` method taking a callback as argument. Something like:

``` js
destroy: callback => {
    global.removeEventListener('resize', resizeListener);
    callback();
}
```